graphql_compiler/tests/test_data_tools/data_tool.py

```diff
@@ -258,20 +258,25 @@ def init_sql_integration_test_backends():
             engine.execution_options(isolation_level='AUTOCOMMIT').execute(
                 text('CREATE DATABASE db_2;'))
 
+            engine.execution_options(isolation_level='AUTOCOMMIT').execute(
```

Comment:
All of the code below is a hack fix.

Reply:
What makes them hacks? Can we fix them now? If not, might it be helpful to add a TODO?

Reply:
Sorry, I meant to say that I was removing a hack hat already existed. I could see how that wasn't clear based on what I said.

Reply:
Ohhhh I misunderstood - gotcha! Thank you for fixing the hacks 🥇